File: wronggou-shop-web/src/main/resources/static/js/bTypeTemplateService.js
app.service('bTypeTemplateService',function($http){

    this.findPage=function(page, rows){
        return  $http.get('http://localhost:8087/bTypeTemplate/findPage?page=' + page + '&rows=' + rows);
    }
    this.save=function(entity){
        return   $http.post('http://localhost:8087/bTypeTemplate/add',entity);
    }
    this.findOne=function(id){
        return $http.get('http://localhost:8087/bTypeTemplate/findOne?id='+id);
    }
    this.update=function(entity){
        return   $http.post('http://localhost:8087/bTypeTemplate/update',entity);
    }
    this.delete=function(selectIds){
        return  $http.get('http://localhost:8087/bTypeTemplate/delete?ids='+selectIds);
    }
    this.findSpecList=function(id){
        return $http.get('http://localhost:8087/bTypeTemplate/findSpecList?id='+id);
    }
});

File: wronggou-sellergoods-interfaces/src/main/java/com/wrongo/service/bTypeTemplateService.java
package com.wrongo.service;

import com.wronggo.model.PageResult;
import com.wronggo.model.Specification;
import com.wronggo.model.TbTypeTemplate;

import java.util.List;
import java.util.Map;

public interface bTypeTemplateService {
    List<TbTypeTemplate> selectall();
    public PageResult findPage(int pageNum, int pageSize);
    public void   insert(TbTypeTemplate tbTypeTemplate );
    public void  update(TbTypeTemplate tbTypeTemplate);
    TbTypeTemplate selectbyid(Long  id);
    public void  delete(Long [] ids);
    public List<Map> findSpecList(Long id);
}
